src/misc/lv_anim.c

```diff
@@ -333,7 +335,6 @@ int32_t lv_anim_path_custom_bezier3(const lv_anim_t * a)
 /**********************
  *   STATIC FUNCTIONS
  **********************/
-#include <stdio.h>
```

Comment:
Ah, it was just  left over from debugging, and stdio.h is not required in this file.